### DISCUSSION TITLE
招待メールクリック後にGoogleログインを試そうとした場合，パスワード設定に失敗しかつ招待リンクが使用不能になる

### DISCUSSION THREAD

**Discussion by akhtkm** [topotal/waroom-roadmap#18] [Others]
以下の事象に遭遇したのでフィードバックします．

事象

1. 招待メールリンクからユーザー作成画面に遷移した（正常）
2. パスワード設定画面が表示されたが，Googleアカウントログインのボタンがあったためそちらを選択
3. ユーザー作成画面ではなく組織作成画面に遷移した
4. ユーザー作成が完了せず，再度招待リンクをクリックして，パスワード設定を試そうと思ったがリンクがエラーで使用不能になった
  エラーメッセージ：`400 invitation not found or already used.`
5. 管理者からは `User already exists` になっているが，パスワード設定ができず，ログインができない．
6. パスワード再発行を試すも，メールが届かない（迷惑メールボックスにも配布されていない）

提案？
- Googleアカウントでの招待リンクが使用できる
or
- 招待リンクからはGoogleアカウントログインをさせない（後から連携できるようにする）

**Comment by sawa-zen** [comment]
@akhtkm 
フィードバックありがとうございます。ご不便おかけしてしまい申し訳有りません。急ぎ確認いたします 🙇🏻

**Comment by sawa-zen** [comment]
@akhtkm 
おそらく手順 3 まででユーザーは正しく作成されております。
以下のフローを一度試していただけますか？
1. ログインボタンからログイン画面へ遷移
1. 組織名を入力
1. [頂いたコメント](https://github.com/topotal/waroom-roadmap/discussions/18#discussion-4614402) の 2 の手順で選択した Google のアカウントを同様に選択してログイン

**Reply by akhtkm** [reply]
ありがとうございます！いただいた内容で無事ログイン（Googleログイン）ができました！ @sawa-zen

**Reply by sawa-zen** [reply]
@akhtkm 
いえいえ！アカウント作成後の遷移先や作成成功のフィードバックが甘くお客様を混乱させていた事がわかったので大変助かりました 🙇🏻
こちら急ぎ修正しようと思います！ 
今後ともよろしくおねがいたします！
